### ISSUE TITLE
<bug> Opera GX actively refuses connection

### ISSUE THREAD

**Issue by shootybangman** [manucabral/YoutubeMusicRPC#32]
**Describe the bug**
The program initializes, syncs, and detects my browser just fine. But the log states that remote debugging is not enabled, so it opens Opera GX with some tabs, then returns with "<urlopen error [WinError 10061] No connection could be made because the target machine actively refused it>". This is most likely a problem with Opera GX and not the program, but I couldn't find anything helpful on remote debugging, so I thought I'd see if anyone here can help me out. I've tried closing all tabs, and closing out of Opera GX entirely.

**Expected behavior**
Program should successfully open and connect to web browser.

**Screenshots**
![image](https://github.com/manucabral/YoutubeMusicRPC/assets/78578466/017395f2-3036-4abd-b794-bfc1a85fb97b)

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 11, Home, 64-bit

**Comment by manucabral** [comment]
Hi, to establish a connection with the application, you need to close all instances of your OperaGX browser first. Once you've done that, open YoutubeMusicRPC; the application will instantly open your browser (OperaGX) and you can use it as usual, try it.

**Comment by shootybangman** [comment]
I have tried this, but it still seems to not work. It seems that the application attempts to open Opera GX, but still fails to, even though it is closed. If I have Opera GX open while the app runs, it opens a new window with some tabs, with the URLs "software/opera" and "stable/[username]", and gives me the same error. Could there possibly be something in setup that I typed incorrectly? I wasn't quite sure what most of the terminology in the setup meant, so there could have been something that I misinterpreted, possibly causing this error.

**Comment by manucabral** [comment]
> I have tried this, but it still seems to not work. It seems that the application attempts to open Opera GX, but still fails to, even though it is closed. If I have Opera GX open while the app runs, it opens a new window with some tabs, with the URLs "software/opera" and "stable/[username]", and gives me the same error. Could there possibly be something in setup that I typed incorrectly? I wasn't quite sure what most of the terminology in the setup meant, so there could have been something that I misinterpreted, possibly causing this error.

You can safely delete the 'settings.json' configuration file. During the setup, when prompted for the Profile Name, simply press Enter. On another note, if this doesn't resolve the issue, I'm currently testing the program on the latest version of Opera GX  to provide you with a solution.

**Comment by manucabral** [comment]
![test2](https://github.com/manucabral/YoutubeMusicRPC/assets/83357673/c8775547-e361-4b55-9adc-c6c05e4f483d)

**Comment by shootybangman** [comment]
Yeah, even with all of my tabs closed and with Opera GX closed out entirely, the app does not open it back up and gives me the same error. The new window with those tabs only opens up if Opera GX is open, but it gives me the error either way. Changing the profile name to "default" (not entering anything when prompted) did not help either. I appreciate your help.

**Comment by manucabral** [comment]
Please check issue (#26) and try that version maybe helps you.

Ref:
> It same like me, but work fine while using version 0.5.1 my opera GX version LVL 5 (core: 102.0.4880.117) [client.log](https://github.com/manucabral/YoutubeMusicRPC/files/13246284/client.log)

**Comment by shootybangman** [comment]
That works! Thanks a lot.
